frontend/src/layout/navigation/MainNavigation.tsx

```diff
@@ -47,30 +47,35 @@ interface MenuItemProps {
     icon: JSX.Element
     identifier: string
     to: string
+    highlight?: boolean
     onClick?: () => void
 }
 
-const MenuItem = ({ title, icon, identifier, to, onClick }: MenuItemProps): JSX.Element => {
+const MenuItem = ({ title, icon, identifier, to, highlight = false, onClick }: MenuItemProps): JSX.Element => {
     const { scene, loadingScene } = useValues(sceneLogic)
     const { collapseMenu } = useActions(navigationLogic)
 
-    function activeScene(): string {
-        const nominalScene = loadingScene || scene
-        // Scenes with special handling can go here
-        return sceneOverride[nominalScene] || nominalScene
-    }
-
     function handleClick(): void {
         onClick?.()
         collapseMenu()
     }
 
+    const className: string = useMemo(() => {
```

Comment:
This could be slightly cleaner using `classnames` module, however don't seem to be using it.

This is messy code. Suggestion: let's extract it as a function underneath the class?

Reply:
Rolled back to previous approach.